behat.yml.dist

```diff
@@ -59,6 +60,18 @@ default:
                 firefox:
                     selenium2:
                         browser: firefox
+                panther:
+                    panther:
+                        manager_options:
+                            connection_timeout_in_ms: 5000
+                            request_timeout_in_ms: 120000
+                            chromedriver_arguments:
+                                - --log-path=etc/build/chromedriver.log
+                                - --verbose
+                            capabilities:
+                                acceptSslCerts: true
+                                acceptInsecureCerts: true
+                                unexpectedAlertBehaviour: accept
```

Comment:
```suggestion
                    panther:
                        options:
                            browser: "chrome"
                            webServerDir: "%paths.base%/public"
                            external_base_uri: "https://127.0.0.1:8080/"
                        kernel_options:
                            APP_ENV: test_cached
                            APP_DEBUG: false
                        manager_options:
                            connection_timeout_in_ms: 5000
                            request_timeout_in_ms: 120000
                            capabilities:
                                browserName: chrome
                                browser: chrome
                                extra_capabilities:
                                    chromeOptions:
                                        args:
                                            - "--window-size=2880,1800"
                                            - "--headless"
                                            - "start-fullscreen"
                                            - "start-maximized"
                                            - "no-sandbox"
                                acceptSslCerts: true
                                acceptInsecureCerts: true
                                unexpectedAlertBehaviour: accept
```

I spent time a year ago reading the code of the extension and how to minimize and make it works. Here is what I ended up configuring in one my Sylius plugin to make Panther works as intented.
The part about `kernel_options` is maybe skippable because I think it was a bug that has been resolved since then.